lib/events/dynamic.go

```diff
@@ -37,7 +47,16 @@ func FromEventFields(fields EventFields) (events.AuditEvent, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	eventType := fields.GetString(EventType)
+	getFieldEmpty := func(field string) string {
+		i, ok := fields[field]
+		if !ok || !isInterfaceString(i) {
+			return ""
+		}
+
+		return i.(string)
+	}
```

Comment:
If you're going to do `i.(string)` anyway, you don't need a type switch.

```suggestion
	getFieldEmpty := func(field string) string {
		i, ok := fields[field]
		if !ok {
			return ""
		}
		s, ok := i.(string)
		if !ok {
			return ""
		}

		return s
	}
```

Reply:
Actually, you really can do:
```
s, _ := i.(string)
return s
```